unlock-protocol.com/src/static/_redirects

```diff
@@ -1,5 +0,0 @@
-# This file is custom to netlify and used to perform redirects and URL rewrites
```

Comment:
This file is only useful for fully dynamic paths where we cannot statically generate the pages.
It is not used here because we know of all posts when deploying the site.